test/gtest/uct/cuda/test_switch_cuda_device.cc

```diff
@@ -272,3 +272,60 @@ UCS_TEST_P(test_another_thread, detect_mem_type_cuda_fabric)
 #endif
 
 _UCT_MD_INSTANTIATE_TEST_CASE(test_another_thread, cuda_cpy);
+
+class test_p2p_no_current_cuda_ctx : public uct_p2p_rma_test {
+public:
+    void test_xfer(send_func_t send, size_t length, unsigned flags,
+                   ucs_memory_type_t mem_type) override;
+};
+
+void test_p2p_no_current_cuda_ctx::test_xfer(send_func_t send, size_t length,
```

Comment:
maybe rename to test_xfer_on_thread or similar?

Reply:
Renamed.